receiver/mysqlreceiver/metadata.yaml

```diff
@@ -552,6 +552,33 @@ metrics:
       monotonic: true
       aggregation_temporality: cumulative
     attributes: [cache_status]
+  mysql.qcache.hits:
```

Comment:
Is there a "misses" value which could be part of this metric as well? (differentiated from "hits" by an attribute)

Reply:
Unfortunately, there are no misses value part of this metric as far as I can glean from [this](https://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Qcache_hits). And


+-------------------------+------------+
| Variable_name           | Value      |
+-------------------------+------------+
| Qcache_free_blocks      | 1          |
| Qcache_free_memory      | 1048557680 |
| Qcache_hits             | 0          |
| Qcache_inserts          | 0          |
| Qcache_lowmem_prunes    | 0          |
| Qcache_not_cached       | 1          |
| Qcache_queries_in_cache | 0          |
| Qcache_total_blocks     | 1          |
+-------------------------+------------+